examples/zenohpico/z_queryable.c

```diff
@@ -0,0 +1,114 @@
+//
```

Comment:
These are Zenoh-Pico examples.
Why do we have them in Zenoh-CPP as well?

Reply:
see above